.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": false
+}
```

Comment:
@peterjc I just caught that in the most recent set of fixes. It seems that if I hit "ignore" on one of VSCode's popups, it adds a setting to remember my choice. Sorry about that! I will remove it.  It is pretty simple to check for it before committing and get rid of it.